scripts/run-main-analyzer.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import argparse
+import os
+
+from assayist.processor.main_analyzer import MainAnalyzer
+
+parser = argparse.ArgumentParser(
+    description='Download the artifacts associated with a build and unpack them')
+parser.add_argument('--input-dir', type=str,
+                    help='The diretory containing the "metadata" directory')
+args = parser.parse_args()
+
+input_dir = args.input_dir or '.'
+
+input_dir = os.path.join(input_dir, 'metadata')
+
+MainAnalyzer().main(input_dir)
```

Comment:
ATM, the `MainAnalyzer` expect two input files. *buildinfo.json* and *taskinfo.json* We can get just one os them from message bus. And I think we will switch from topic `koji.task.closed` to `koji.build.complete` thus we will have just file *buildinfo.json*

It would be good to update main analyzer to remove requirement for input file *taskinfo.json* or to fetch it in processor.

Reply:
Both are already fetched in the download-unpack-build.py script. The only thing we expect to get from the message is the build id. If there is no taskinfo.json then the Component will not be created since we don't know what the build_type is, but there are no other negative repercussions. If there is a better way to handle missing taskinfo.json then we can definitely update that, but it should not block production use of the analyzer.